bundles/automation/org.eclipse.smarthome.automation.module.core.test/src/test/groovy/org/eclipse/smarthome/automation/module/core/internal/RuntimeRuleTest.groovy

```diff
@@ -203,6 +206,43 @@ class RuntimeRuleTest extends OSGiTest{
     }
     
     @Test
+    public void 'assert that timerTrigger works'(){
+        def testExpression = "* * * * * ?" 
+        def testItemName = "myLampItem5"
+
+        def triggerConfig = [cronExpression:testExpression]
+        def triggers = [new Trigger("MyTimerTrigger", "TimerTrigger", triggerConfig)] 
+       
+        def actionConfig = [itemName:testItemName, command:"ON"]
+        def actions = [new Action("MyItemPostCommandAction", "ItemPostCommandAction", actionConfig, null)]
+        
+        def conditionConfig = [operator:"=", itemName:testItemName, state:"OFF"]
+        def conditions = [new Condition("MyItemStateCondition", "ItemStateCondition", conditionConfig, null)]
+             
+        def rule = new Rule("MyRule"+new Random().nextInt(),triggers, conditions, actions, null, null)
+        rule.name="MyTimerTriggerTestRule"
+        logger.info("Rule created: "+rule.getUID())
+        
+        def ItemRegistry itemRegistry = getService(ItemRegistry)
+        def SwitchItem lampItem = itemRegistry.getItem(testItemName)
+        lampItem.send(OnOffType.OFF); 
+        waitForAssert({
+            assertThat lampItem.state,is(OnOffType.OFF)
+        })
+  
+        def ruleRegistry = getService(RuleRegistry) as RuleRegistry
+        ruleRegistry.add(rule)
+        ruleRegistry.setEnabled(rule.UID, true)
+        waitForAssert({
+          assertThat ruleRegistry.getStatus(rule.UID).status, is(RuleStatus.IDLE)
+        })
+        
+        waitForAssert({  
+            assertThat lampItem.state,is(OnOffType.ON)
+        })
```

Comment:
Would be great to test at least a second cycle: means check if the rule is triggered again and again?